pkg/server/serverpb/status.proto

```diff
@@ -1185,6 +1209,14 @@ service Status {
     };
   }
 
+  // PGWireCancelQuery cancels a SQL query given its pgwire "secret ID".
+  rpc PGWireCancelQuery(PGWireCancelQueryRequest) returns (PGWireCancelQueryResponse) {
+    option (google.api.http) = {
```

Comment:
i got some pusback on making this available as a http endpoint and to remove it in a different PR, wonder if you'll be told the same

Reply:
Ah interesting, let us see what people think.. I put it here since the other CancelQuery endpoint is in here, but open to hearing alternatives.